code/game/objects/items/melee/swords.dm

```diff
@@ -0,0 +1,298 @@
+	icon_state = broken_icon
+	inhand_icon_state = broken_icon
+	worn_icon_state = broken_icon
+	update_appearance()
+	playsound(user, 'sound/effects/structure_stress/pop3.ogg', 100, TRUE)
+	force -= 5
+	wound_bonus = 1
+	throw_range = 2
+	embedding = list("embed_chance" = 10, "impact_pain_mult" = 15)//jagged metal in wound would be more painful.
+	block_chance = 20
+	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/melee/sword/rust/gold
+	name = "rusty gilded broadsword"
+	desc = "A sharp steel forged sword. It's got a rich guard and pommel. It's edge is rusty and corroded."
+	icon_state = "broadsword_gold_rust"
+	inhand_icon_state = "broadsword_gold"
+	worn_icon_state = "broadsword_gold"
+	broken_icon = "broadsword_gold_broken"
+
+/obj/item/melee/sword/rust/claymore
+	name = "rusty claymore"
+	desc = "A rusted claymore, it smells damp and it has seen better days."
+	icon_state = "claymore_rust"
+	inhand_icon_state = "claymore"
+	worn_icon_state = "claymore"
+	broken_icon = "claymore_broken"
+
+/obj/item/melee/sword/rust/claymoregold
+	name = "rusty holy claymore"
+	desc = "A weapon fit for a crusade... or it used to be..."
+	icon_state = "claymore_gold_rust"
+	inhand_icon_state = "claymore_gold"
+	worn_icon_state = "claymore_gold"
+	broken_icon = "claymore_gold_broken"
+
+/obj/item/melee/sword/rust/cultblade
+	name = "rusty dark blade"
+	desc = "Once used by worshipers of forbidden gods, now its covered in old rust."
+	icon_state = "cultblade_rust"
+	inhand_icon_state = "cultblade_rust"
+	broken_icon = "cultblade_broken"
+
+/obj/item/melee/sword/claymore
+	name = "holy claymore"
+	desc = "A weapon fit for a crusade! It lacks a holy shine however."
+	force = 18
+	icon_state = "claymore_gold"
+	inhand_icon_state = "claymore_gold"
+	worn_icon_state = "claymore_gold"
+
+/obj/item/melee/sword/claymore/darkblade
+	name = "dark blade"
+	desc = "Spread the glory of the dark gods! Even if they don't bless this blade."
+	icon_state = "cultblade"
+	inhand_icon_state = "cultblade"
+	worn_icon_state = "cultblade"
+	lefthand_file = 'icons/mob/inhands/64x64_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/64x64_righthand.dmi'
+	inhand_x_dimension = 64
+	inhand_y_dimension = 64
+
+/obj/item/melee/sword/reforged
+	name = "Reforged longsword"
+	desc = "A hard steel blade, it's edge has been forged to be incredibly strong. It feels light."
+	icon_state = "reforged"
+	inhand_icon_state = "reforged"
+	worn_icon_state = "reforged"
+	force = 25
+	throwforce = 15
+	throw_range = 6
+	block_chance = 40
+	wound_bonus = 5
+	armour_penetration = 15
+	embedding = list("embed_chance" = 30, "impact_pain_mult" = 10)
+
+/obj/item/melee/sword/reforged/shitty
+	var/broken = FALSE
+	var/rustiness = 0 //This isn't a mistake, this causes it to break instantly upon use.
+	var/broken_icon = "reforged_broken"
+
+/obj/item/melee/sword/reforged/shitty/afterattack(target, mob/user, proximity_flag)
+	. = ..()
+	if(broken)
+		return ..()
+	if(ismovable(target))
+		decrease_uses(user)
```

Comment:
I know you're gonna have trouble with this but if you're gonna copy+paste this code across two types it really should be a component.